docs/components/best-practices/modeling/choosing-the-resource-binding-type.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Choosing the resource binding type
+tags:
+  - BPMN
+description: "Understand the differences between 'latest' and 'deployment' binding for linked resources"
+---
+
```

Comment:
I like starting pages with a value proposition to answer "why should I care?"
```suggestion
Camunda offers version binding for linked processes, decisions, or forms so you can deploy new versions without disrupting live processes and prevent production outages.
```

Reply:
Ah, I just noticed that you meant this to be the first sentence of the page, not the `description` tag.